src/app/routes/utils/regex/index.js

```diff
@@ -56,5 +57,9 @@ export const secondaryColumnDataRegexPath = getSecondaryColumnDataRegex(
   allServices,
 );
 
+export const recommendationsDataRegex = getRecommendationsDataRegex(
+  allServices,
```

Comment:
should this be all services ? can we limit to only services with the `hasStoryRecommendations` flag ?

Reply:
All this is used for is the routing of the fixture data files for the local development server. Not sure if we need to go out of our way to do a check for the flag.